packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/stepSimulation.st

```diff
@@ -1,4 +1,7 @@
 simulation
 stepSimulation
+	"Decreases the startNode level by the specified output, increases the endNode level by the specified intake (for every connection) and steps up the simulationTime."
```

Comment:
Please update comment to match the functionality of the message: This method just calls performSimulationStep on every connection, the behaviour of which may change based on its type.